src/api/election.ts

```diff
@@ -0,0 +1,35 @@
+import { Election } from '@votingworks/ballot-encoder'
+import {
+  DeleteElectionResponse,
+  GetElectionResponse,
+  PutElectionResponse,
+} from '../config/types'
+import fetchJSON from '../util/fetchJSON'
+
+export async function get(): Promise<Election | undefined> {
+  return fetchJSON<GetElectionResponse>('/election')
```

Comment:
suggest prefixing as `/config/election` or the like. We can stuff a few different things  in`/config` over time.

Reply:
Done